migrations/env.py

```diff
@@ -14,7 +14,8 @@
 
 # Interpret the config file for Python logging.
 # This line sets up loggers basically.
-fileConfig(config.config_file_name)
+if not context.config.attributes.get("unit-tests", False):
+    fileConfig(config.config_file_name)
```

Comment:
How is this actually used 🤔. Do we use special Alembic config during tests?

Reply:
Added a comment